README.md

```diff
@@ -1,128 +1,247 @@
 dp-healthcheck
-================
+==============
 
-A health check git repository for DP
+A health check git repository for Digital Publishing that implements the [Health Check Specification](https://github.com/ONSdigital/dp/blob/master/standards/HEALTH_CHECK_SPECIFICATION.md).  All Digital Publishing apps must implement a health check using this library.  Functions that implement the `Checker` type are registered with the library which will check internal and external measures of the apps health.  The library will then call these functions periodically to determine the overall health of the app and report this back using the included handler.
 
-### Getting started
+Getting started
+---------------
 
-Read the [Health Check Specification](https://github.com/ONSdigital/dp/blob/master/standards/HEALTH_CHECK_SPECIFICATION.md) for details.
+* [Add health check to an app](#adding-a-health-check-to-an-app)
+* [Implementing a `Checker` function](#implementing-a-checker)
 
-#### How to use
-1. Add Health Check library to an app
+Adding a health check to an app
+-------------------------------
 
-2. Create an array of Health Check clients by calling `NewClient()` passing in the following:
+1. Import the library and your HTTP server dependencies:
 
-- An optional RCHTTP clienter; if none is provided one will be created
-- A function that implements the `Checker` interface
+    ```
+    package main
 
-3. Call `Create()` passing in the following:
-
-- Versioning Information
-- Critical time duration; time to wait for dependent apps critical unhealthy status to make current app unhealthy- Time Interval to run health checks on dependencies
-- Clients; An array of clients created in the previous step, like so:
+    import (
+        ...
+        health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+        "github.com/ONSdigital/go-ns/server"
+        "github.com/gorilla/mux"
+        ...
+    )
+    ```
 
-```
-package main
+2. Create a version object defining the version of your app:
 
-import (
+    ```
     ...
-    health "github.com/ONSdigital/dp-healthcheck/healthcheck"
-    ...
-)
-...
-var BuildTime, GitCommit, Version string
-...
 
-func main() {
+    var BuildTime, GitCommit, Version string
+    const criticalTimeout = time.Minute
+    const interval = 10 * time.Second
     ...
 
-    criticalTimeout := time.Minute
-    interval := 10 * time.Second
+    func main() {
+        ctx := context.Context(context.Background())
+        ...
 
-    versionInfo := health.CreateVersionInfo(
-        time.Unix(BuildTime, 0),
-        GitCommit,
-        Version,
-    )
+        versionInfo := health.CreateVersionInfo(
+            time.Unix(BuildTime, 0),
```

Comment:
BuildTime is a string not int64, should this line change so that we pass in the string value of BuildTime now the `CreateVersionInfo` func handles thee parsing of this field?